src/scenes/Wallet/components/EncryptDialog/index.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Dialog, { DialogTitle, DialogContent, DialogActions } from 'material-ui/Dialog';
+import { TextField, Button } from 'material-ui';
+import { withStyles } from 'material-ui/styles';
+import { FormattedMessage, injectIntl, intlShape, defineMessages } from 'react-intl';
+import _ from 'lodash';
+
+import styles from './styles';
+import appActions from '../../../../redux/App/actions';
+
+
+const messages = defineMessages({
+  passphrase: {
+    id: 'encrypt.passphrase',
+    defaultMessage: 'Passphrse',
+  },
+});
+
+@injectIntl
+@withStyles(styles, { withTheme: true })
```

Comment:
if we're not using `classes` in here, please remove:
1. `withStyles` and `styles` imports
2. remove the `withStyles` decorator
3. remove `styles.js` file